sections/earn/ClaimTab/ClaimTab.tsx

```diff
@@ -97,6 +99,32 @@ const ClaimTab: React.FC<ClaimTabProps> = ({ tradingRewards, stakingRewards, tot
 			? blockExplorerInstance.txLink(txHash)
 			: undefined;
 
+	const fetchFeePeriodData = useCallback(async () => {
+		if (!isL2) return;
+		try {
+			const {
+				contracts: { FeePool },
+			} = synthetix.js!;
+			const [feePeriodDuration, recentFeePeriods] = await Promise.all([
+				FeePool.feePeriodDuration(),
+				FeePool.recentFeePeriods(0),
+			]);
+
+			const now = Math.ceil(new Date().getTime() / 1000);
+			const startTime = Number(recentFeePeriods.startTime);
+			const duration = Number(feePeriodDuration);
+
+			setIsCloseFeePeriodEnabled(now > duration + startTime);
```

Comment:
in this function you should also fetch the gas limit estimate for the transaction to close the fee period and set it to state so that the est. price will show up here 

![image](https://user-images.githubusercontent.com/39738607/114813716-0537f100-9df6-11eb-8637-37455e937e10.png)

Reply:
```
const gasLimit = await synthetix.getGasEstimateForTransaction({
				txArgs: [],
				method: FeePool.estimateGas.closeCurrentFeePeriod,
			});


setGasLimitEstimate(gasLimit)
```

Reply:
No because GasLimit is hardcoded in L2 for now :(